content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Adding Windows worker nodes
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+This page explains how to add Windows worker nodes to a kubeadm cluster.
+
+## {{% heading "prerequisites" %}}
+
+* A running [Windows Server 2019](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing)
```

Comment:
We should probably point to WS2022, as 2019 is in extended support

Reply:
should i replace the text in squire brackets to say `[WIndows Server 2022]` instead of 2019?